src/components/ErrorBoundary/ErrorBoundary.jsx

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
+
+class ErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  componentDidCatch() {
+    this.setState({ hasError: true });
```

Comment:
Consider logging the `error` and `info` parameters inside `componentDidCatch` for better error tracking and debugging.